apps/wso2-digital-wallet/src/pages/ReceiveCoins/ReceiveCoins.css

```
@@ -7,284 +7,318 @@
+  background: #ffffff;
+  border: 1.5px solid #E5E0D8;
+  border-radius: 16px;
+  padding: 14px 16px;
 }
 
-.amount-input-container {
-  width: 100%;
-  max-width: 400px;
-  margin-bottom: 32px;
+.receive-address-card-text {
+  font-family: 'JetBrains Mono', 'Monaco', 'Menlo', 'Ubuntu Mono', monospace;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)